ParkingZoneApp/Services/Services.cs

```diff
@@ -27,12 +27,12 @@ public void Remove(T value)
             _repository.Delete(value);
         }
 
-        public IEnumerable<T> GetAll()
+        public virtual IEnumerable<T> GetAll()
```

Comment:
```suggestion
        public IEnumerable<T> GetAll()
```

Reply:
why did you made it virtual?

Reply:
I was overriding this method then I decided to create new version of this method then I forgot to remove virtual keyword